software/moltres/moltres.md

```diff
@@ -16,15 +16,15 @@ conditions used to construct the Moltres governing equations can be found on the
 full-fledged Moltres input file can be found
 [here](./wiki/input_example/). New Moltres
 users who have never used MOOSE before are encouraged to check-out its
-[wiki](http://mooseframework.org/wiki/),
-[tutorials](http://mooseframework.org/wiki/MooseTutorials/), and
-[examples](http://mooseframework.org/wiki/MooseExamples/) to help understand the
+[wiki](http://mooseframework.org/documentation.menu.html),
```

Comment:
There currently is not a new working MOOSE wiki and I am not convinced that linking to the documentation is sufficient. @smpark7 @robfairh thoughts?

Reply:
also @katyhuff thoughts?

Reply:
This is quite sufficient..